src/libs/actions/Report.js

```diff
@@ -442,6 +442,10 @@ function fetchOrCreateChatReport(participants) {
     })
 
         .then((data) => {
+            if (data.jsonCode !== 200) {
+                alert(data.message);
```

Comment:
Should we `return`? We are still making a call to `Get` `reportStuff` with no reportID ?

Reply:
Yes, my mistake.